etna/models/holt_winters.py

```diff
@@ -207,6 +213,12 @@ def fit(self, df: pd.DataFrame, regressors: List[str]) -> "_HoltWintersAdapter":
         :
             Fitted model
         """
+        freq = pd.infer_freq(df["timestamp"], warn=False)
```

Comment:
May be we cat create a separate method for this, if it is used in several models